_includes/manuals/1.10/4.4.8.3_prov_windows.md

```diff
@@ -0,0 +1,160 @@
+Provisioning Windows is a two step process. The first step, creating Installation Media, is not discussed here but can be found in the [Wiki](http://projects.theforeman.org/projects/foreman/wiki/Tutorials). The second step, setting up templates, is discussed in the following section.
```

Comment:
The wiki page has yet to be added. A preview, [the existing version](https://github.com/helge000/wimaging-ng/blob/master/doc/wimaging.md)